samples/ingestion/ingestion-client/.pre-commit-config.yaml

```diff
@@ -0,0 +1,12 @@
+repos:
+  - repo: local
+    hooks:
+      - id: bicep
+        name: bicep
+        description: Lint and build Bicep files
+        entry: ./samples/ingestion/ingestion-client/scripts/generate_arm_templates.sh
```

Comment:
Will the pre-commit work for this on Windows? Or will this be devcontainer only?

Reply:
pre commit worked only for devcontainer. I tried adding it for Windows but could not find a solution. Update the PR to fail the build in case the bicep is not in sync with the json.

Reply:
Normally I agree with you that we should have a `scripts` folder, but seeing as the main OS is Windows, do you think we should keep `generate_arm_templates.sh` in the `infra` folder tidied away rather than leaving a loan script in the `scripts` folder?